app/src/main/java/nl/mpcjanssen/simpletask/Simpletask.kt

```diff
@@ -196,21 +196,18 @@ class Simpletask : ThemedNoActionBarActivity() {
     }
 
     override fun onSearchRequested(): Boolean {
-        if (options_menu == null) {
-            return false
+        options_menu?.let {
+            val searchMenuItem = it.findItem(R.id.search)
+            MenuItemCompat.expandActionView(searchMenuItem)
+            return true
         }
-        val searchMenuItem = options_menu!!.findItem(R.id.search)
-        MenuItemCompat.expandActionView(searchMenuItem)
-
-        return true
+        return false
     }
```

Comment:
The `if ($VAR == null) return` approach results in code that is very order-sensitive. If we ever refactor this bit of code so it no longer comes after a null check, we introduce the possibility of NPEs. Same for `if ($VAR != null) { block() }`, since `block()` will still work fine if copied somewhere outside of the null check.

The updated version is impossible to *accidentally* break in this way, since `it` will throw a compiler error if used in the wrong scope, and becomes a nice block of "keep this together if possible". Since I expect to do a bunch of refactoring soon, these are nice qualities to have.

This type of change is the majority of changes in the set; I'll just mark the rest by commenting "Brittle".